src/txkube/_network.py

```diff
@@ -224,6 +245,11 @@ def __repr__(self):
 
 
 
+def log_result(document, action):
```

Comment:
This is pretty self explanatory, but a docstring that made it clear this was for use with eliot might be nice.